### PR TITLE
Add language support for Vietnamese language

### PR DESCRIPTION
**Pull request by PlayerNguyen**
This PR follows the issue #223

### CODE REVIEW

**Comment by PlayerNguyen** [comment]
Here is some screenshot under Vietnamese UI

image
image

File: src/i18n/vi/index.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	isCurrentVersionLatest: 'Bạn đang sử dụng phiên bản mới nhất',
+	isLatestVersion: 'Có phiên bản mới hơn',
+	knowledge: 'Kiến thức',
+	knowledgeSaved: 'Đã lưu kiến thức',
+	language: 'Ngôn ngữ',
+	lastUsedModels: 'Các mô hình đã sử dụng gần đây',
+	light: 'Giao diện Sáng',
+	lowVram: 'VRAM thấp',
+	mainGpu: 'GPU chính',
+	// messageOfTheDay: 'Thông điệp trong ngày',
```
Comment (fmaclen) [inline src/i18n/vi/index.ts:64]: Let's remove the empty line, and uncomment (or remove) line 64.

**Review by fmaclen — CHANGES_REQUESTED** [review]
Thanks for submitting this PR, I think it mostly looks good 👍 
Just left you a minor comment below that needs fixing.

**Comment by PlayerNguyen** [comment]
@fmaclen I changed it and we are good to go

**Review by fmaclen — APPROVED** [review]
This is great, thanks @PlayerNguyen! 🚀

**Pull request merged**